perfkitbenchmarker/benchmark_spec.py

```diff
@@ -229,7 +229,19 @@ def ConstructVirtualMachines(self):
 
         if DISK_SPEC in group_spec:
           disk_spec_class = disk.GetDiskSpecClass(cloud)
-          disk_spec = disk_spec_class(**group_spec[DISK_SPEC][cloud])
+          disk_spec_dict = group_spec[DISK_SPEC][cloud]
+          # We have to translate the data_disk_type flag here because
```

Comment:
Add a note here to indicate that data_disk_type flag may contain legacy values that were copied from the scratch_disk_type flag at the beginning of the run.